File: customer-portal-frontend/src/app/modules/order/_models/ShipToAddressUpdateDto.ts
export interface ShipToAddressUpdateDto {
    ShipToName: string;
    ShipToPhone: string;
    ShipToAddress1: string;
    ShipToAddress2: string | null;
    ShipToCity: string;
    ShipToState: string;
    ShipToZip: string;
    ShipToCountry: string;
}

File: customer-portal-backend/src/Lumio.CustomerPortal.Services/Order/Dtos/ShipToAddressUpdateDto.cs
namespace Lumio.CustomerPortal.Services.Order
{
    public class ShipToAddressUpdateDto
    {
        public string ShipToName { get; set; }
        public string ShipToPhone { get; set; }
        public string ShipToAddress1 { get; set; }
        public string? ShipToAddress2 { get; set; }
        public string ShipToCity { get; set; }
        public string ShipToState { get; set; }
        public string ShipToZip { get; set; }
        public string ShipToCountry { get; set; }
    }
}
